go.mod

```diff
@@ -52,8 +52,8 @@ require (
 	github.com/tonistiigi/fsutil v0.0.0-20191018213012-0f039a052ca1 // indirect
 	go.uber.org/zap v1.11.0 // indirect
 	golang.org/x/sys v0.0.0-20191127021746-63cb32ae39b2
-	k8s.io/api v0.0.0
-	k8s.io/apimachinery v0.0.0
+	k8s.io/api v0.17.0
+	k8s.io/apimachinery v0.17.0
```

Comment:
TODO - need to clean up our k8s.io references. Upstream fixed their tagging so we should be able to drop a lot of the replace stanzas.